### ISSUE TITLE
Error | Reference to #31

### ISSUE THREAD

**Issue by DPG896** [BlockHorizons/PerWorldPlayer#32]
The whole crash shows here:
https://crash.pmmp.io/view/6313791

**Comment by DPG896** [comment]
Whole Backtrace
```
Backtrace:
#0 plugins/PerWorldPlayer (1).phar/src/BlockHorizons/PerWorldPlayer/player/PlayerManager(34): pocketmine\errorhandler\ErrorToExceptionHandler::handle(integer 2, string[21] Undefined array key 1, string[112] phar:///home/container/plugins/PerWorldPlayer (1).phar/src/BlockHorizons/PerWorl, integer 34)
#1 plugins/PerWorldPlayer (1).phar/src/BlockHorizons/PerWorldPlayer/world/WorldInstance(88): BlockHorizons\PerWorldPlayer\player\PlayerManager->get(object pocketmine\player\Player#34592)
#2 plugins/PerWorldPlayer (1).phar/src/BlockHorizons/PerWorldPlayer/world/WorldInstance(65): BlockHorizons\PerWorldPlayer\world\WorldInstance->save(object pocketmine\player\Player#34592, object BlockHorizons\PerWorldPlayer\world\data\PlayerWorldData#105681, integer 1)
#3 plugins/PerWorldPlayer (1).phar/src/BlockHorizons/PerWorldPlayer/world/WorldListener(65): BlockHorizons\PerWorldPlayer\world\WorldInstance->onPlayerExit(object pocketmine\player\Player#34592, NULL , true)
#4 pmsrc/src/event/RegisteredListener(75): BlockHorizons\PerWorldPlayer\world\WorldListener->onPlayerQuit(object pocketmine\event\player\PlayerQuitEvent#105723)
#5 pmsrc/src/event/Event(62): pocketmine\event\RegisteredListener->callEvent(object pocketmine\event\player\PlayerQuitEvent#105723)
#6 pmsrc/src/player/Player(2088): pocketmine\event\Event->call()
#7 pmsrc/src/network/mcpe/NetworkSession(581): pocketmine\player\Player->onPostDisconnect(string[17] client disconnect, NULL )
#8 pmsrc/src/network/mcpe/NetworkSession(514): pocketmine\network\mcpe\NetworkSession->pocketmine\network\mcpe\{closure}()
#9 pmsrc/src/network/mcpe/NetworkSession(583): pocketmine\network\mcpe\NetworkSession->tryDisconnect(object Closure#105687, string[17] client disconnect)
#10 pmsrc/src/network/mcpe/raklib/RakLibInterface(151): pocketmine\network\mcpe\NetworkSession->onClientDisconnect(string[17] client disconnect)
#11 pmsrc/vendor/pocketmine/raklib-ipc/src/RakLibToUserThreadMessageReceiver(75): pocketmine\network\mcpe\raklib\RakLibInterface->onClientDisconnect(integer 0, string[17] client disconnect)
#12 pmsrc/src/network/mcpe/raklib/RakLibInterface(122): raklib\server\ipc\RakLibToUserThreadMessageReceiver->handle(object pocketmine\network\mcpe\raklib\RakLibInterface#32961)
#13 pmsrc/vendor/pocketmine/snooze/src/SleeperHandler(123): pocketmine\network\mcpe\raklib\RakLibInterface->pocketmine\network\mcpe\raklib\{closure}()
#14 pmsrc/vendor/pocketmine/snooze/src/SleeperHandler(82): pocketmine\snooze\SleeperHandler->processNotifications()
#15 pmsrc/src/Server(1660): pocketmine\snooze\SleeperHandler->sleepUntil(double 1648587622.9755)
#16 pmsrc/src/Server(1046): pocketmine\Server->tickProcessor()
#17 pmsrc/src/PocketMine(304): pocketmine\Server->__construct(object BaseClassLoader#2, object pocketmine\utils\MainLogger#3, string[16] /home/container/, string[24] /home/container/plugins/)
#18 pmsrc/src/PocketMine(327): pocketmine\server()
#19 pmsrc(11): require(string[60] phar:///home/container/PocketMine-MP.phar/src/PocketMine.php)
```

**Comment by Muqsit** [comment]
Addressed in [Dev #36](https://poggit.pmmp.io/babs/1f626)
